JobConfig/digitize/prolog.fcl

```diff
@@ -74,6 +74,19 @@ Digitize: {
       module_type : Prescaler
       prescaleFactor : 1 # by default, don't prescale
       prescaleOffset : 0
+    }
+     TriggerableCH : {
```

Comment:
would it make sense to `@table::...` the previous `Triggerable` module instance? Also, a minor thing... any strong reason for not including the pions to the `particleTypes` list?